assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -356,6 +364,23 @@ export default function ArchivePicker({view = ADMIN_VIEW}) {
     });
   };
 
+  const setCustomFeaturedImage = async imgId => {
+    await wp.data.dispatch('core/editor').editPost({featured_media: imgId});
+    dispatch({type: ACTIONS.CLOSE_SIDEBAR});
+    document.querySelector('.media-modal-icon').click();
```

Comment:
Same remark as previous comment on similar code, this selector is not specific enough